ci/deps/travis-37.yaml

```diff
@@ -20,5 +20,6 @@ dependencies:
   - s3fs
   - isort
   - pip
+  - pyreadstat
```

Comment:
can you also add to environment.yml (note there is a PR changing it as we speak, so might have a conflict)

Reply:
Add it via conda or pip (conda has the package only for Python 3.6 and 3.7, but not for 3.5)?

Reply:
I've used pip because I guess we want that to work on Python 3.5.